### ISSUE TITLE
Problem with "accept="audio/mp3" in Safari - Version 11.0.3 (13604.5.6)

### ISSUE THREAD

**Issue by nazarposhta** [react-dropzone/react-dropzone#607]
Hi all. I found a bug with accept="audio/mp3" in Safari. This type of props looks like not supported in this browser. When I'm trying to add *.mp3 file with this parameter - method "onDropRejected" just drops me an error. I even tried to set all of ".mp3" aliases like 
`audio/mp3, audio/mpeg3, audio/mpg, audio/x-mp3, audio/x-mpeg, audio/x-mpeg3, audio/x-mpg`
but it still doesn't work. At the moment is only one way - to remove "accept" at all. But, would be good to filter it because some who by accident can add something else.
Any ideas?

**Comment by Mickyfen17** [comment]
Hey @nazarposhta, I just ran into this issue today, it seems to also be happening in Firefox. 
The way I got around it was by using `accept: "audio/mp3, audio/mpeg"`.
It looks like Safari and Firefox use the MIME `audio/mpeg` whilst Chrome prefers `audio/mp3`.

This post helped me https://stackoverflow.com/questions/10688588/which-mime-type-should-i-use-for-mp3

Hopefully this will help you, if this issue still exists for you.
